Applications/PlazaDetails/ServiceMaturity.qmd

```diff
@@ -4,34 +4,35 @@ aliases:
 - 'Applications/PlazaDetails/ServiceMaturity.html'
 ---
 
-All the services on the openEO algorithm plaza are assigned a maturity level that indicates what users can expect from the service in regard to its performance and metadata. Currently, we have five different maturity levels for each service, namely Prototype being the primary and default level, followed by Incubating, Verified, Validated and Operational as advanced services. These levels are determined solely based on software readiness and user documentation criteria. These criteria are generally designed to ensure that the service meets specific standards and provides customers with a certain level of quality.
+Each service in the openEO algorithm plaza is assigned a maturity level that indicates its qualitative documentation and performance. Currently, five different maturity levels for each service are available. The prototype is the primary and default level, followed by incubating, verifying, validating, and operating as advanced services. These levels are determined solely based on software readiness and user documentation criteria. These criteria are generally designed to ensure that the service meets specific standards and provides customers with a certain level of quality.
+
 
 
 ## Level 1: Prototype
 
-By default, every published service will have a prototype level. It is expected that users consider the following points when publishing a service:
+By default, every published service has a prototype level. It is expected that users consider the following points when publishing a service:
 
 - The service is executable, and basic logging information is supported.
 
 - A possible reference or a general overview of what it tries to implement is provided as service metadata.
 
-:::{callout-note}
+::: {.callout-note}
 
-If your service satisfies the criteria for a higher level, you can request an upgrade anytime once your service is published. Nevertheless, please note that every criterion must be satisfied for an upgrade.
+Suppose a service meets the criteria for a higher level. In that case, an upgrade can be requested anytime after publication. It's essential to ensure that all requirements are satisfied before the upgrade can be approved.
 
 :::
 
 ## Level 2: Incubating
 
-In addition to the criteria for prototype level, a few additional criteria, as mentioned below, need to be satisfied to be upgraded to incubating service.
+In addition to the criteria for prototype level, a service needs to meet the following requirements to be an incubating service:
```

Comment:
I think stage is more appropriate than level?

needs to meet the following requirements to reach the incubating stage